lib/scenic/theme.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Scenic.Theme do
```

Comment:
I'm 100% in favor of having separate `%Theme{}` and theme manager modules. We have a few options for the naming scheme:
- `Theme` and `ThemeManager`
- `Theme` and `Theme.Manager`
- `Themes.Theme` and `Themes.Manager` (i.e. Phoenix context style)

I have an inclination for the first or third but I don't have a super strong feeling.

Reply:
I thought about this a bit more, and kind of doubling-down on the Phoenix convention, `Theme` and `Themes` might be the way to go.